packages/preview/ensias-report/0.1.0/lib.typ

```diff
@@ -0,0 +1,178 @@
+#let IMAGE_BOX_MAX_WIDTH = 120pt
+#let IMAGE_BOX_MAX_HEIGHT = 50pt
+
+#let project(title: "", subtitle: none, school-logo: "images/ENSIAS.svg", company-logo: none, authors: (), mentors: (), jury: (), branch: none, academic-year: none, french: false, body) = {
+  // Set the document's basic properties.
+  set document(author: authors, title: title)
+  set page(
+    numbering: "1",
+    number-align: center,
+    footer: locate(loc => {
+      // Omit page number on the first page
+      let page-number = counter(page).at(loc).at(0);
+      if page-number > 1 {
+        line(length: 100%, stroke: 0.5pt)
+        v(-2pt)
+        text(size: 12pt, weight: "regular")[
+          ENSIAS
+          #h(1fr)
+          #page-number
+          #h(1fr)
+          #academic-year
+        ]
+      }
+    })
+  )
+
+  let dict = json("resources/i18n/en.json")
+  let lang = "en"
+  if french {
+    dict = json("resources/i18n/fr.json")
+    lang = "fr"
+  }
```

Comment:
i like the idea to import i18n ressources over json files. I'll copy that into my template. thanks

Reply:
Sure, go ahead :grin: